packages/react-core/src/components/AboutModal/__tests__/AboutModal.test.tsx

```diff
@@ -15,11 +15,13 @@ const props: AboutModalProps = {
 };
 
 describe('AboutModal', () => {
-  test('closes with escape', () => {
+  test('closes with escape', async () => {
+    const user = userEvent.setup();
+    
     render(<AboutModal {...props} isOpen />);
 
-    userEvent.type(screen.getByRole('dialog'), '{esc}');
-    expect(props.onClose).toHaveBeenCalled();
+    await user.type(screen.getByRole('dialog'), '{Escape}');
+    waitFor(() => expect(props.onClose).toHaveBeenCalled());
```

Comment:
`waitFor` returns a promise which should be awaited. There are several other `waitFor` usages in this change that should be awaited as well.